tests/test_manageiq_provider.py

```diff
@@ -86,7 +88,7 @@
 }
 
 GET_RETURN_VALUES = {
-    'openshift_without_hawkular': {
+    'openshift_without_metrics': {
```

Comment:
consider s/openshift_without_metrics/openshift_without_monitoring/g.

more importantly, there is `miq_ansible_module.params = {'metrics': True}` on L274